plugin-template

```diff
@@ -345,27 +415,34 @@ def check_for_deprecated_files(plugin_root_dir, sections):
 def migrate_github_issues(config, plugin_root_dir, verbose=False):
     """Moves the plugin's issue checker CI/CD off of Redmine and onto GitHub"""
     print("Updating template_config.yml and templating ci section")
-    name = config['plugin_name']
-    redmine = config['redmine_project']
+    name = config["plugin_name"]
+    redmine = config["redmine_project"]
     if not redmine:
-        print("Redmine project name not set. Please set 'redmine_project' in template_config.yml"
-              " and rerun this command")
+        print(
+            "Redmine project name not set. Please set 'redmine_project' in template_config.yml"
+            " and rerun this command"
+        )
         return 2
-    config['issue_tracker'] = 'github'
-    config['redmine_project'] = None
-    config['update_redmine'] = False
-    config.pop('config')
-    config['config'] = config.copy()
+    config["issue_tracker"] = "github"
+    config["redmine_project"] = None
+    config["update_redmine"] = False
+    config.pop("config")
+    config["config"] = config.copy()
     write_template_section(config, "generate_config", plugin_root_dir, verbose=verbose)
     write_template_section(config, "github", plugin_root_dir, verbose=verbose)
     print("Updating pyproject.toml to point to GitHub issues\n")
     search_string = "pulp.plan.io"
-    replace_string = f"github.com\/pulp\/{name}"
+    replace_string = f"github.com/pulp/{name}"
```

Comment:
flake8 alerted on this change and I'm assuming the '\' aren't supposed to be there?

```
>>> name = "catdog"
>>> f"github.com\/pulp\/{name}"
'github.com\\/pulp\\/catdog'
```

Reply:
@gerrod3 is the above what you expect? It looks like python interprets \/ as \\\/ in f-strings.

Maybe I should just convert it to a normal string and use `format()`?

Reply:
You can try. I have no idea if it'll work. I think the problem is that the string you pass in has to be a sed format string, but python does weird things with the slashes and that's why I had an escape slash there.